pkg/storaged/storage.scss

```
@@ -626,19 +626,20 @@ td.storage-action {
 
 /* Overview side panels */
 
-#storage .storage-sidebar table {
-    table-layout: fixed;
+.sidepanel-row {
+    padding-left: var(--pf-global--spacer--md);
+    padding-right: var(--pf-global--spacer--md);
+    padding-top: var(--pf-global--spacer--sm);
+    padding-bottom: var(--pf-global--spacer--sm);
```

This is better with shorthand:

```suggestion
    padding: var(--pf-global--spacer--sm) var(--pf-global--spacer--md);
```

---

Done!

---

Explanation:

CSS has shorthand to combine things, it takes the form of one of the following:

- analogue clock directions: top right bottom left (this is why it has funky ordering... just think of the hands moving around a clock and you'll be fine)
- top/bottom left/right (what I'm using above)
- top sides bottom
- every-direction-at-once

It's only useful when defining all the padding-* border-* margin-* etc. at one time. (You can't skip a side, so re-declaring something will overwrite it.)